package.json

```diff
@@ -24,24 +24,15 @@
   },
   "devDependencies": {
     "@reactivex/rxjs": "5.0.0-beta.6",
-    "codecov.io": "0.1.6",
+    "@types/chai": "3.4.*",
+    "@types/es6-shim": "0.0.*",
+    "@types/glob": "5.0.*",
+    "@types/grunt": "0.4.*",
     "dojo-loader": ">=2.0.0-beta.7",
-    "dts-generator": "~1.7.0",
-    "glob": "^7.0.3",
     "grunt": "^1.0.1",
-    "grunt-contrib-clean": "^1.0.0",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-watch": "^1.0.0",
     "grunt-dojo2": ">=2.0.0-beta.16",
-    "grunt-release": "0.13.1",
-    "grunt-text-replace": "0.4.0",
-    "grunt-ts": "^5.5.1",
-    "grunt-tslint": "^3.0.3",
-    "grunt-typings": ">=0.1.5",
     "intern": "~3.2.3",
-    "istanbul": "0.4.3",
-    "remap-istanbul": ">=0.6.4",
-    "tslint": "next",
-    "typescript": "2.0.2"
+    "tslint": "^3.15.1",
+    "typescript": "~2.0.3"
```

Comment:
I suspect 2.1 will introduce breaking changes, contrary to semver... I realised that we should do this with the others too, don't want this accidentally flipping over to 2.1.